src/06-loops.js

```diff
@@ -61,7 +82,12 @@ function rangeEveryOther(min, max) {}
  *  containsWhileLoop([ "left", "up", "right" ], "down");
  *  //> false
  */
-function containsWhileLoop(array, target) {}
+function containsWhileLoop(array, target) {
+  while (array.indexOf(target) !== -1){
```

Comment:
nice work using `.indexOf()` as with all other methods lets make sure we know exactly what this is doing.  What does `.indexOf()` return?  Do we need a loop if we use `.indexOf()`?  How could we write this without `.indexOf()`?